lib/sourcebit.js

```diff
@@ -204,6 +206,15 @@ class Sourcebit {
 
   writeFiles(files) {
     const filesByPath = files.reduce((result, file) => {
+      if (!file.path || typeof file.path !== "string") {
```

Comment:
This isn't related to the code, but a small addition that ended up in this PR. 🙈